content/YubiHSM2/Usage_Guides/YubiHSM_2_for_Microsoft_SQL_Server_Deployment_Guide--Enabling_Always_Encrypted_with_YubiHSM_2/Prerequisites_and_Preparations.adoc

```diff
@@ -0,0 +1,40 @@
+== Prerequisites and Preparations
+
+The audience of this document is an experienced system administrator with a good understanding of Microsoft SQL Server management. In addition, it is helpful to be familiar with the terminology, software and tools specific to YubiHSM 2. As a primer for these, refer to link:Terminology.adoc[the Terminology chapter].
+
+In order to follow the steps provided in this guide, the following prerequisites must be met:
+
+* Microsoft Windows Server 2012 SP2 or higher, with Microsoft .NET Framework 4.7.2 or higher. The operating system should be installed in a secure computer network. The system administrator must also have elevated system privileges.
+
+* Access to Microsoft SQL Server 2017 with SQL Server Management Studio (SSMS) 2017 or higher.
+
+* YubiHSM 2 software and tools for Windows downloaded from link:../Releases[the Yubico YubiHSM 2 Release page] and available on the system to be used.
```

Comment:
this link is broken, it needs an extra ../ at the start

Reply:
5ceaa1777c1aef13c3b49adaaf7606b7c333cbc6